config/overlays/odh-test/configmap/inferenceservice-openshift-ci-llm.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: inferenceservice-config
-  namespace: kserve
+  namespace: opendatahub
```

Comment:
don't we need this for upstream?

Reply:
I mean I think we need this test case for upstream. so don't we need to use `overlay/test` ?

Reply:
Is it for a specific odh? then I was misunderstood. I thought it was a general test for llmd.